File: workout/beecrowd/code/1019.java
import java.io.IOException;
import java.util.Scanner;
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner leitor = new Scanner(System.in);
        int sec = leitor.nextInt();
        
        int horas = sec / 3600;
        int minutos = (sec %3600 ) / 60;
        int segundos = (sec %3600 ) % 60;
        
        System.out.printf("%d:%d:%d\n", horas, minutos, segundos);
        
    }
 
}

File: workout/beecrowd/code/1019.cs
using System;

public class URI
{
    public static void Main(string[] args)
    {
        int total = int.Parse(Console.ReadLine());
        int horas = total / 3600;
        int minutos = (total % 3600 )/ 60;
        int segundos = (total % 3600) % 60;
        
        Console.WriteLine($"{horas}:{minutos}:{segundos}");
    }
}